llama-index-core/llama_index/core/download/pack.py

```diff
@@ -99,7 +106,7 @@ def download_llama_pack_template(
     refresh_cache: bool = False,
     custom_dir: Optional[str] = None,
     custom_path: Optional[str] = None,
-    base_file_name: str = "base.py",
+    base_file_name: str = "__init__.py",
```

Comment:
can use `__init__.py` as base file to install module now since we add the top-level shortcuts in all of our llama-packs (and integrations) when doing the migration for v0.10x